### PR TITLE
Substitute deprecated nvidia-docker

### PR DESCRIPTION
**Pull request by HeAndres**
Fixes NVIDIA/TensorRT-LLM#1095

### CODE REVIEW

**Comment by HeAndres** [comment]
The corrected first command can be seen in the [docs](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/sample-workload.html) for Nvidia Container Toolkit. Installation of NCT is a pre-requisite, as written in the Installation section of the [Readme.md](https://github.com/NVIDIA/TensorRT-LLM/blob/main/README.md).

**Comment by Shixiaowei02** [comment]
Thanks for the suggestion, we will fix this issue.

**Comment by HeAndres** [comment]
I would like to contribute if it is possible. Could you tell me further steps to make a successful PR, or any thing that you need to make this PR successful? Thank you in advance!

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ please run the following commands to install TensorRT-LLM for x86_64 users.
 ```bash
 # Please use the `nvidia-docker` application, using only `docker` may cause exceptions.
```
Comment (Shixiaowei02) [inline README.md]:
```suggestion
```

Comment (Shixiaowei02) [reply README.md:110]: Please delete this sentence. Thanks!

**Review by Shixiaowei02 — CHANGES_REQUESTED** [review]

**Review by Shixiaowei02 — APPROVED** [review]

**Comment by Shixiaowei02** [comment]
> I would like to contribute if it is possible. Could you tell me further steps to make a successful PR, or any thing that you need to make this PR successful? Thank you in advance!

Thank you for your contribution, please wait for the code to be merged.

**Pull request merged**